### PR TITLE
smartystreets-javascript-sdk - allow for batch execution of US street and zipcode lookups

### PR DESCRIPTION
**Pull request by newbish**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## smartystreets-javascript-sdk ([<kbd>unpkg</kbd>](https://unpkg.com/browse/smartystreets-javascript-sdk@latest/))
was missing the following properties:
1. usReverseGeo
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 438c6b4b0c3225694974a0de3e4b8012dd9fb0cc
</p>

File: types/smartystreets-javascript-sdk/smartystreets-javascript-sdk-tests.ts [hunk 1]
```diff
@@ -25,6 +26,24 @@ client
         err, // $ExpectType any
     ) => console.error(err));
 
+// usStreet - Batch
+client = SmartyStreetsSDK.core.buildClient.usStreet(credentials);
+batch = new SmartyStreetsSDK.core.Batch<SmartyStreetsSDK.usStreet.Lookup>();
+lookup = new SmartyStreetsSDK.usStreet.Lookup();
+lookup.street = '4646 Scott St';
```
Comment (joefeser) [inline types/smartystreets-javascript-sdk/smartystreets-javascript-sdk-tests.ts:33]: Is Frenchy's Chicken that good?

**Review by joefeser — APPROVED** [review]

File: types/smartystreets-javascript-sdk/smartystreets-javascript-sdk-tests.ts (hunk 1 above)
Comment (newbish) [reply types/smartystreets-javascript-sdk/smartystreets-javascript-sdk-tests.ts:33]: [Queen B](https://twitter.com/BeyLegion/status/906316723594387456?s=20) approves... no cap.

**Comment by newbish** [comment]
ready to merge

**Pull request merged**